### PR TITLE
Split sim.dat to sim00.dat and sim01.dat

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Summary
to speed up the macOS build

## Impact
build system

## Testing
pass CI

### CODE REVIEW

File: testlist/sim00.dat [hunk 1]
```diff
@@ -1,27 +1,21 @@
...
+-Darwin,sim:lvgl
 -Darwin,sim:nsh2
 -Darwin,sim:nx11
 -Darwin,sim:nxlines
 -Darwin,sim:nxwm
--Darwin,sim:touchscreen
--Darwin,sim:lvgl
-
-# macOS doesn't have ALSA
--Darwin,sim:alsa
```
Comment (v01d) [inline testlist/sim00.dat:23]: this was missed in the split version, which is causing the failure.

Comment (v01d) [reply testlist/sim00.dat:23]: actually i see it is there
maybe the syntax you're using to include configs is not supported?

Comment (xiaoxiang781216) [reply testlist/sim00.dat:23]: testbuild.sh fail to remove the dir from line:
```
dotest $i${line/$dir/}
``
https://github.com/apache/incubator-nuttx/pull/3298 fix it by adding "

Comment (v01d) [reply testlist/sim00.dat:23]: but don't you need the glob so that you can add the configs using `*`?

Comment (xiaoxiang781216) [reply testlist/sim00.dat:23]: config finding need glob:
```
list=`find boards$dir -name defconfig | cut -d'/' -f4,6`
```
but prefix removing don't need:
```
dotest $i${line/$dir/}
```

Comment (v01d) [reply testlist/sim00.dat:23]: got it

**Comment by v01d** [comment]
Build is now 50m, nice.
Can you open the corresponding PRs for nuttx & apps?

**Comment by xiaoxiang781216** [comment]
Done, since this pach break other repo CI, I have to merge the change directly after success.

**Comment by v01d** [comment]
Sorry I missed the other PRs, it was nighttime in my timezone.

**Pull request merged**